view/next-project/src/components/common/Table/Table.tsx

```diff
@@ -68,18 +72,22 @@ const Table: FC<TableContentProps> = (props) => {
           ×
         </button>
       </div>
-      <DataTable
-        columns={props.columns}
-        data={filteredItems}
-        pagination
-        highlightOnHover
-        pointerOnHover
-        onRowClicked={(row) => {
-          if (row.curriculum) router.push(`/${props.route}/${row.curriculum.id}`);
-          else router.push(`/${props.route}/${row.id}`);
-        }}
-        customStyles={CUSTOM_STYLES}
-      />
+      {loader ? (
+        <div>Loading</div>
+      ) : (
+        <DataTable
+          columns={props.columns}
+          data={filteredItems}
+          pagination
+          highlightOnHover
+          pointerOnHover
+          onRowClicked={(row) => {
+            if (row.curriculum) router.push(`/${props.route}/${row.curriculum.id}`);
+            else router.push(`/${props.route}/${row.id}`);
+          }}
+          customStyles={CUSTOM_STYLES}
+        />
+      )}
```

Comment:
https://stackoverflow.com/questions/75068071/react-data-table-component-pagination-error-hydration-failed-because-the-in
これと同じエラーが起きてこう書いたけど、無駄なstateな気がしてる

Reply:
https://github.com/Selman-S/React-Data-Table-Component
このライブラリ、2年前から更新ないやん
リファクタ案件かも～